packages/client/src/views/RecordAudit/ActionMenu.tsx

```diff
@@ -118,9 +115,10 @@ export const ActionMenu: React.FC<{
     <>
       <DropdownMenu id="action">
         <DropdownMenu.Trigger>
-          <PrimaryButton icon={() => <CaretDown />}>
+          <Button type="primary" size="medium" iconPosition="right">
```

Comment:
@jpye-finch The Button component shouldn't really concern itself where the icon is being positioned. I'll remove it and see if I can just replicate the same using flex